lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -399,7 +399,7 @@ void SendProcessEvent(LaunchMethod launch_method) {
 // Grab any STDOUT and STDERR from the process and send it up to VS Code
 // via an "output" event to the "stdout" and "stderr" categories.
 void SendStdOutStdErr(lldb::SBProcess &process) {
-  char buffer[1024];
+  char buffer[4096];
```

Comment:
could you also modify that other buffer and have a constant that can be used by both of them? 4096 should be more than enough for most messages.

After that, this PR should be fine by me.

Reply:
Moved this into a constant.